doc/man3/SSL_write.pod

```diff
@@ -2,12 +2,13 @@
 
 =head1 NAME
 
-SSL_write_ex, SSL_write - write bytes to a TLS/SSL connection
+SSL_write_ex, SSL_write, SSL_sendfile - write bytes to a TLS/SSL connection
 
 =head1 SYNOPSIS
 
  #include <openssl/ssl.h>
 
+ int SSL_sendfile(SSL *s, int fd, off_t offset, size_t size, int flags);
```

Comment:
The meaning of flags here is undocumented

Reply:
Also return type no longer matches the implementation

Reply:
I'll fix the type.
Regarding "flags", as I've mentioned in the commit message it is very OS specific. I wonder how it should be provided here. In Linux there are no sendfile flags, while in FreeBSD are important.
I was thinking of stating this as follows:
"For FreeBSD B<flags> is the same as the flags parameter to sendfile. It is provided here to allow users to maintain the same interface. For Linux B<flags> is ignored."
What do you think? Should I add the **current** FreeBSD flags here?

Reply:
I think it is fine to document these as platform specific with a reference to the sendfile documentation for that platform. I don't think we need to duplicate the FreeBSD flags here.